### DISCUSSION TITLE
Docker Toolbox funciona legal no Windows Server 2012 R2?

### DISCUSSION THREAD

**Discussion by ghost** [frontendbr/forum#1445] [Perguntas e Respostas]
Tem um Servidor Windows Server 2012 R2 em Núvem, preciso trabalhar com o Docker, na documentação : 
https://docs.docker.com/docker-for-windows/install/

Fala que o Docker for Windows funciona somente a partir do 2016, como alternativa tem Docker Toolbox 

Alguém já realizou testes e funcionou bacaninha?

**Comment by adrianoavelino** [comment]
Eu uso ele no Windows 7 Professional e funcionou legal nos meus estudos e testes aqui no trabalho. A diferença é que não acesso os container por localhost, mas por um IP de uma VM do VirtualBox fornecido pelo próprio Docker Toolbox.

**Comment by ghost** [comment]
Ok obrigado vou instalar e realizar alguns testes em meu servidor

**Comment by lucasctd** [comment]
@wburlani Tu não vai usar isso em produção né?
Eu pensei em pegar um VPS Windows Server 2016 pq queria usar o Azure DevOPS (Docker) no meu host.
Ao pesquisar o que mais encontrei foi gente desaconselhando pois mesmo o Docker CE, ainda roda os containers linux em VM (Hyper V) no Windows.

**Comment by ghost** [comment]
Como meu ambiente em nuvem é apenas a versão 12 (ambiente corporativo), e para atualizar para 16 terei complicações internas, então vou ter que testa-lo mesmo no 12 se rodar Aleluia se não paciência

**Reply by tecnicacbm** [reply]
Amigo, você consegui usar no Windows server 2012

**Reply by carlosblanc0** [reply]
Nao se tem a opção de instalar um ubuntu? Rodar docker desse jeito é meio que um hack job, na minha humilde opinião.

**Comment by carlosblanc0** [comment]
Se o seu provedor cloud não suporta [nested virtualization](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/nested-virtualization), eu acredito que sera importável de rodar nessa versão do windows.
